stix_shifter_modules/msatp/configuration/config.json

```diff
@@ -39,6 +39,18 @@
             "clientSecret": {
                 "type": "password"
             }
+        },
+        "includeAlerts":{
+            "type": "boolean",
+            "default": true
+        },
+        "includeHostOs": {
+            "type": "boolean",
+            "default": true
+        },
+        "includeNetworkInfo": {
+            "type": "boolean",
+            "default": true
```

Comment:
i would make the default `false`. User can decide whether they want to use these features.

Reply:
good idea